src/js/reducers.js

```diff
@@ -434,6 +434,9 @@ function ui(state = {}, action) {
                 case "NON-MEDIA":
                     app.displayLayout = "nonmedia"
                     break
+                case "WEB_VIEW":
+                    app.displayLayout = "webview"
```

Comment:
I think web-view would be more inline with the naming conventions, except nonmedia is kind of ruining my claim.

Reply:
updated in https://github.com/smartdevicelink/generic_hmi/pull/291/commits/89d551f877e2ce71bde1d3f7092c3ff225a8acf6

If we changed nonmedia we could fix [this special case](https://github.com/smartdevicelink/generic_hmi/pull/291/files#diff-63ab2bf602f05ad65cee8459f425cc94R52) and just use .toLower() for the registerApp call.